.github/workflows/kernel-upload.yml

```diff
@@ -0,0 +1,43 @@
+---
+name: Upload kernel source RPM
+on:
+  push:
+    branches:
+      - main
```

Comment:
You mentioned an update to one of the scripts wasn't triggering the pipeline after changing the target branch - may need to add that new branch here to trigger on push

Reply:
Not sure if it was related to this yml, but could be a similar issue with one of the others

Reply:
Thanks, that makes sense! I can test the `*-upload.yml` files by using a non-main branch by temporarily changing it. For everything else, pushing to a non-main branch should work as-is. I'll try this today.